common/src/main/java/com/wynntils/wynn/model/quests/QuestInfo.java

```diff
@@ -121,6 +121,29 @@ public boolean isMiniQuest() {
         return isMiniQuest;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || (!(o instanceof QuestInfo questInfo))) return false;
+
+        // Consider it the same quest just based on "constant" aspects of the quest
+        if (level != questInfo.level) return false;
+        if (isMiniQuest != questInfo.isMiniQuest) return false;
+        if (!name.equals(questInfo.name)) return false;
+        if (length != questInfo.length) return false;
+        return additionalRequirements.equals(questInfo.additionalRequirements);
```

Comment:
Sorry for the trouble, but this still not the same as other code parts. I may have mislead you though.

This is the format I use.
<img width="496" alt="image" src="https://user-images.githubusercontent.com/49001742/208138874-c0214af3-a5a8-43c0-9c30-19aa472473d8.png">

Reply:
Thanks. (I used the IntelliJ default, but tweaked it to allow for the new pattern-matching instanceof)

I don't know how much we need to be consistent on this sorts of things, but I guess it does not hurt. What is more important however is to be sure to actually *do* implement equals and hashCode, where needed.